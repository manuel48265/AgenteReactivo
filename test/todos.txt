mapas/mapa30.map 1 0 6 6 1
mapas/mapa50.map 1 0 3 3 0
mapas/mapa75.map 1 0 60 11 0
mapas/mapa30.map 1 0 3 3 0
mapas/mapa30.map 1 0 3 3 1
mapas/mapa30.map 1 0 3 3 3
mapas/mapa30.map 1 0 25 11 0
mapas/mapa30.map 1 0 25 11 1
mapas/mapa30.map 1 0 25 11 3
mapas/mapa30.map 1 0 22 11 0
mapas/mapa30.map 1 0 22 11 1
mapas/mapa30.map 1 0 22 11 3
mapas/mapa30.map 1 0 8 14 0
mapas/mapa30.map 1 0 8 14 1
mapas/mapa30.map 1 0 8 14 3
mapas/mapa30.map 1 0 7 26 0
mapas/mapa30.map 1 0 7 26 1
mapas/mapa30.map 1 0 7 26 3
mapas/mapa50.map 1 0 42 19 0
mapas/mapa50.map 1 0 42 19 2
mapas/mapa50.map 1 0 42 19 4
mapas/mapa50.map 1 0 42 19 5
mapas/mapa50.map 1 0 7 9 0
mapas/mapa50.map 1 0 7 9 2
mapas/mapa50.map 1 0 7 9 4
mapas/mapa50.map 1 0 7 9 5
mapas/mapa50.map 1 0 9 42 0
mapas/mapa50.map 1 0 9 42 2
mapas/mapa50.map 1 0 9 42 4
mapas/mapa50.map 1 0 9 42 5
mapas/mapa75.map 1 0 60 11 0
mapas/mapa75.map 1 0 60 11 1
mapas/mapa75.map 1 0 60 11 3
mapas/mapa75.map 1 0 60 11 5
mapas/mapa75.map 1 0 7 9 0
mapas/mapa75.map 1 0 7 9 1
mapas/mapa75.map 1 0 7 9 3
mapas/mapa75.map 1 0 7 9 5
mapas/mapa75.map 1 0 9 42 0
mapas/mapa75.map 1 0 9 42 1
mapas/mapa75.map 1 0 9 42 3
mapas/mapa75.map 1 0 9 42 5
mapas/picto.map 1 0 71 71 3
mapas/picto.map 1 0 61 13 3
mapas/picto.map 1 0 20 47 3
mapas/cuadricula.map 1 0 50 49 7
mapas/medieval.map 1 0 87 52 1
mapas/medieval.map 1 0 65 74 1
mapas/medieval.map 1 0 20 77 1
mapas/mapa100.map 1 0 3 3 3
mapas/mapa100.map 1 0 76 40 3
mapas/mapa100.map 1 0 70 30 3
mapas/vertigo.map 1 0 17 45 1
mapas/vertigo.map 1 0 17 45 3
mapas/vertigo.map 1 0 17 45 5
mapas/vertigo.map 1 0 92 91 1
mapas/vertigo.map 1 0 92 91 3
mapas/vertigo.map 1 0 92 91 5
mapas/vertigo.map 1 0 92 91 7
mapas/vertigo.map 10 0 92 91 7
mapas/vertigo.map 28 0 92 91 7
mapas/vertigo.map 31 0 92 91 7
mapas/vertigo.map 1723 0 92 91 7
mapas/paldea1.map 1 0 22 55 2
mapas/paldea1.map 1 0 22 55 7
mapas/paldea1.map 1 0 70 47 2
mapas/paldea1.map 1 0 70 47 7
mapas/paldea1.map 1 0 91 45 2
mapas/paldea1.map 1 0 91 45 7
mapas/luminaliaZA1.map 1 0 50 50 4
mapas/luminaliaZA1.map 1 0 50 50 5
mapas/luminaliaZA1.map 1 0 96 3 4
mapas/luminaliaZA1.map 1 0 96 3 5
mapas/luminaliaZA1.map 1 0 30 72 4
mapas/luminaliaZA1.map 1 0 30 72 5
mapas/mapa30.map 1 1 6 6 1
mapas/mapa50.map 1 1 3 3 0
mapas/mapa75.map 1 1 60 11 0
mapas/mapa30.map 1 1 3 3 0
mapas/mapa30.map 1 1 25 11 0
mapas/mapa30.map 1 1 22 11 0
mapas/mapa30.map 1 1 8 14 0
mapas/mapa30.map 1 1 7 26 0
mapas/mapa50.map 1 1 42 19 0
mapas/mapa50.map 1 1 7 9 0
mapas/mapa50.map 1 1 9 42 0
mapas/mapa75.map 1 1 60 11 0
mapas/mapa75.map 1 1 7 9 0
mapas/mapa75.map 1 1 9 42 0
mapas/picto.map 1 1 71 71 0
mapas/picto.map 1 1 61 13 0
mapas/picto.map 1 1 20 47 0
mapas/cuadricula.map 1 1 50 49 0
mapas/medieval.map 1 1 87 52 0
mapas/medieval.map 1 1 65 74 0
mapas/medieval.map 1 1 20 77 0
mapas/mapa100.map 1 1 3 3 0
mapas/mapa100.map 1 1 76 40 0
mapas/mapa100.map 1 1 70 30 0
mapas/vertigo.map 1 1 17 45 0
mapas/vertigo.map 1 1 92 91 0
mapas/vertigo.map 1 1 92 91 7
mapas/vertigo.map 10 1 92 91 7
mapas/vertigo.map 28 1 92 91 7
mapas/vertigo.map 31 1 92 91 7
mapas/vertigo.map 1723 1 92 91 7
mapas/paldea1.map 1 1 22 55 0
mapas/paldea1.map 1 1 70 47 0
mapas/paldea1.map 1 1 91 45 0
mapas/luminaliaZA1.map 1 1 50 50 0
mapas/luminaliaZA1.map 1 1 96 3 0
mapas/luminaliaZA1.map 1 1 30 72 0
mapas/mapa30.map 1 2 6 6 1
mapas/mapa50.map 1 2 3 3 0
mapas/mapa75.map 1 2 60 11 0
mapas/mapa30.map 1 2 3 3 0
mapas/mapa30.map 1 2 25 11 0
mapas/mapa30.map 1 2 22 11 0
mapas/mapa30.map 1 2 8 14 0
mapas/mapa30.map 1 2 7 26 0
mapas/mapa50.map 1 2 42 19 0
mapas/mapa50.map 1 2 7 9 0
mapas/mapa50.map 1 2 9 42 0
mapas/mapa75.map 1 2 60 11 0
mapas/mapa75.map 1 2 7 9 0
mapas/mapa75.map 1 2 9 42 0
mapas/picto.map 1 2 71 71 0
mapas/picto.map 1 2 61 13 0
mapas/picto.map 1 2 20 47 0
mapas/cuadricula.map 1 2 50 49 0
mapas/medieval.map 1 2 87 52 0
mapas/medieval.map 1 2 65 74 0
mapas/medieval.map 1 2 20 77 0
mapas/mapa100.map 1 2 3 3 0
mapas/mapa100.map 1 2 76 40 0
mapas/mapa100.map 1 2 70 30 0
mapas/vertigo.map 1 2 17 45 0
mapas/vertigo.map 1 2 92 91 0
mapas/vertigo.map 1 2 92 91 7
mapas/vertigo.map 10 2 92 91 7
mapas/vertigo.map 28 2 92 91 7
mapas/vertigo.map 31 2 92 91 7
mapas/vertigo.map 1723 2 92 91 7
mapas/paldea1.map 1 2 22 55 0
mapas/paldea1.map 1 2 70 47 0
mapas/paldea1.map 1 2 91 45 0
mapas/luminaliaZA1.map 1 2 50 50 0
mapas/luminaliaZA1.map 1 2 96 3 0
mapas/luminaliaZA1.map 1 2 30 72 0
mapas/mapa30.map 1 3 6 6 1
mapas/mapa50.map 1 3 3 3 0
mapas/mapa75.map 1 3 60 11 0
mapas/mapa30.map 1 3 3 3 0
mapas/mapa30.map 1 3 3 3 1
mapas/mapa30.map 1 3 3 3 3
mapas/mapa30.map 1 3 25 11 0
mapas/mapa30.map 1 3 25 11 1
mapas/mapa30.map 1 3 25 11 3
mapas/mapa30.map 1 3 22 11 0
mapas/mapa30.map 1 3 22 11 1
mapas/mapa30.map 1 3 22 11 3
mapas/mapa30.map 1 3 8 14 0
mapas/mapa30.map 1 3 8 14 1
mapas/mapa30.map 1 3 8 14 3
mapas/mapa30.map 1 3 7 26 0
mapas/mapa30.map 1 3 7 26 1
mapas/mapa30.map 1 3 7 26 3
mapas/mapa50.map 1 3 42 19 0
mapas/mapa50.map 1 3 42 19 2
mapas/mapa50.map 1 3 42 19 4
mapas/mapa50.map 1 3 42 19 5
mapas/mapa50.map 1 3 7 9 0
mapas/mapa50.map 1 3 7 9 2
mapas/mapa50.map 1 3 7 9 4
mapas/mapa50.map 1 3 7 9 5
mapas/mapa50.map 1 3 9 42 0
mapas/mapa50.map 1 3 9 42 2
mapas/mapa50.map 1 3 9 42 4
mapas/mapa50.map 1 3 9 42 5
mapas/mapa75.map 1 3 60 11 0
mapas/mapa75.map 1 3 60 11 1
mapas/mapa75.map 1 3 60 11 3
mapas/mapa75.map 1 3 60 11 5
mapas/mapa75.map 1 3 7 9 0
mapas/mapa75.map 1 3 7 9 1
mapas/mapa75.map 1 3 7 9 3
mapas/mapa75.map 1 3 7 9 5
mapas/mapa75.map 1 3 9 42 0
mapas/mapa75.map 1 3 9 42 1
mapas/mapa75.map 1 3 9 42 3
mapas/mapa75.map 1 3 9 42 5
mapas/picto.map 1 3 71 71 3
mapas/picto.map 1 3 61 13 3
mapas/picto.map 1 3 20 47 3
mapas/cuadricula.map 1 3 50 49 7
mapas/medieval.map 1 3 87 52 1
mapas/medieval.map 1 3 65 74 1
mapas/medieval.map 1 3 20 77 1
mapas/mapa100.map 1 3 3 3 3
mapas/mapa100.map 1 3 76 40 3
mapas/mapa100.map 1 3 70 30 3
mapas/vertigo.map 1 3 17 45 1
mapas/vertigo.map 1 3 17 45 3
mapas/vertigo.map 1 3 17 45 5
mapas/vertigo.map 1 3 92 91 1
mapas/vertigo.map 1 3 92 91 3
mapas/vertigo.map 1 3 92 91 5
mapas/vertigo.map 1 3 92 91 7
mapas/vertigo.map 10 3 92 91 7
mapas/vertigo.map 28 3 92 91 7
mapas/vertigo.map 31 3 92 91 7
mapas/vertigo.map 1723 3 92 91 7
mapas/paldea1.map 1 3 22 55 2
mapas/paldea1.map 1 3 22 55 7
mapas/paldea1.map 1 3 70 47 2
mapas/paldea1.map 1 3 70 47 7
mapas/paldea1.map 1 3 91 45 2
mapas/paldea1.map 1 3 91 45 7
mapas/luminaliaZA1.map 1 3 50 50 4
mapas/luminaliaZA1.map 1 3 50 50 5
mapas/luminaliaZA1.map 1 3 96 3 4
mapas/luminaliaZA1.map 1 3 96 3 5
mapas/luminaliaZA1.map 1 3 30 72 4
mapas/luminaliaZA1.map 1 3 30 72 5
