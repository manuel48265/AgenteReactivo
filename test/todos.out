         #      Éxito          %     Parada        Val       Nota
======================================================================
         1               100.00          󰥔          S     100.00
         2               100.00          󰥔          S     100.00
         3               100.00          󰥔          S     100.00
         4               100.00          󰥔          S     100.00
         5               100.00          󰥔          S     100.00
         6               100.00          󰥔          S     100.00
         7               100.00          󰥔          S     100.00
         8               100.00          󰥔          S     100.00
         9               100.00          󰥔          S     100.00
        10               100.00          󰥔          S     100.00
        11               100.00          󰥔          S     100.00
        12               100.00          󰥔          S     100.00
        13               100.00          󰥔          S     100.00
        14               100.00          󰥔          S     100.00
        15               100.00          󰥔          S     100.00
        16               100.00          󰥔          S     100.00
        17               100.00          󰥔          S     100.00
        18               100.00          󰥔          S     100.00
        19               100.00          󰥔          S     100.00
        20               100.00          󰥔          S     100.00
        21               100.00          󰥔          S     100.00
        22               100.00          󰥔          S     100.00
        23               100.00          󰥔          S     100.00
        24               100.00          󰥔          S     100.00
        25               100.00          󰥔          S     100.00
        26               100.00          󰥔          S     100.00
        27               100.00          󰥔          S     100.00
        28               100.00          󰥔          S     100.00
        29               100.00          󰥔          S     100.00
        30               100.00          󰥔          S     100.00
        31               100.00          󰥔          S     100.00
        32               100.00          󰥔          S     100.00
        33               100.00          󰥔          S     100.00
        34               100.00          󰥔          S     100.00
        35               100.00          󰥔          S     100.00
        36               100.00          󰥔          S     100.00
        37               100.00          󰥔          S     100.00
        38               100.00          󰥔          S     100.00
        39               100.00          󰥔          S     100.00
        40               100.00          󰥔          S     100.00
        41               100.00          󰥔          S     100.00
        42               100.00          󰥔          S     100.00
        43                90.03          󰥔          S     100.00
        44                90.52          󰥔          S     100.00
        45                90.10          󰥔          S     100.00
        46                98.92          󰁹          S     100.00
        47                99.75          󰥔          S     100.00
        48                99.65          󰥔          S     100.00
        49                99.53          󰥔          S     100.00
        50                99.57          󰥔          S     100.00
        51                99.84          󰥔          S     100.00
        52                99.84          󰥔          S     100.00
        53                93.64          󰁹          S     100.00
        54                93.64          󰁹          S     100.00
        55                93.10          󰁹          S     100.00
        56                93.65          󰁹          S     100.00
        57                93.65          󰁹          S     100.00
        58                93.80          󰁹          S     100.00
        59                93.65          󰁹          S     100.00
        60                93.65          󰁹          S     100.00
        61                93.65          󰁹          S     100.00
        62                93.65          󰁹          S     100.00
        63                93.65          󰁹          S     100.00
        64                94.00          󰁹          S     100.00
        65                94.12          󰁹          S     100.00
        66                94.47          󰁹          S     100.00
        67                94.49          󰁹          S     100.00
        68                94.33          󰁹          S     100.00
        69                94.44          󰁹          S     100.00
        70                97.82          󰥔          S     100.00
        71                97.63          󰥔          S     100.00
        72                97.58          󰥔          S     100.00
        73                97.58          󰥔          S     100.00
        74                97.99          󰥔          S     100.00
        75                97.99          󰥔          S     100.00
        76               100.00          󰥔          S     100.00
        77               100.00          󰥔          S     100.00
        78               100.00          󰥔          S     100.00
        79               100.00          󰥔          S     100.00
        80               100.00          󰥔          S     100.00
        81               100.00          󰥔          S     100.00
        82               100.00          󰥔          S     100.00
        83               100.00          󰥔          S     100.00
        84               100.00          󰥔          S     100.00
        85               100.00          󰥔          S     100.00
        86               100.00          󰥔          S     100.00
        87               100.00          󰥔          S     100.00
        88               100.00          󰥔          S     100.00
        89               100.00          󰥔          S     100.00
        90                90.52          󰥔          S     100.00
        91                89.85          󰥔          S     100.00
        92                90.01          󰥔          S     100.00
        93                99.14          󰁹          S     100.00
        94                99.73          󰥔          S     100.00
        95                99.49          󰥔          S     100.00
        96                99.53          󰥔          S     100.00
        97                99.99          󰥔          S     100.00
        98                99.93          󰥔          S     100.00
        99               100.00          󰥔          S     100.00
       100                94.39          󰁹          S     100.00
       101                91.75          󰁹          S     100.00
       102                91.75          󰁹          S     100.00
       103                91.75          󰁹          S     100.00
       104                91.75          󰁹          S     100.00
       105                91.75          󰁹          S     100.00
       106                91.75          󰁹          S     100.00
       107                93.99          󰁹          S     100.00
       108                94.10          󰥔          S     100.00
       109                94.06          󰥔          S     100.00
       110                97.56          󰥔          S     100.00
       111                97.13          󰥔          S     100.00
       112                98.01          󰥔          S     100.00
       113               100.00          󰥔          S     100.00
       114               100.00          󰥔          S     100.00
       115               100.00          󰥔          S     100.00
       116               100.00          󰥔          S     100.00
       117               100.00          󰥔          S     100.00
       118               100.00          󰥔          S     100.00
       119               100.00          󰥔          S     100.00
       120               100.00          󰥔          S     100.00
       121               100.00          󰥔          S     100.00
       122               100.00          󰥔          S     100.00
       123               100.00          󰥔          S     100.00
       124               100.00          󰥔          S     100.00
       125               100.00          󰥔          S     100.00
       126               100.00          󰥔          S     100.00
       127                89.72          󰥔          S     100.00
       128                90.24          󰥔          S     100.00
       129                90.67          󰥔          S     100.00
       130                98.38          󰁹          S     100.00
       131                99.80          󰥔          S     100.00
       132                98.99          󰥔          S     100.00
       133                99.74          󰥔          S     100.00
       134                99.95          󰥔          S     100.00
       135                99.87          󰥔          S     100.00
       136                99.97          󰥔          S     100.00
       137                92.36          󰁹          S     100.00
       138                92.87          󰁹          S     100.00
       139                92.87          󰁹          S     100.00
       140                92.12          󰁹          S     100.00
       141                90.78          󰁹          S     100.00
       142                89.71          󰁹          S     100.00
       143                94.24          󰁹          S     100.00
       144                93.95          󰁹          S     100.00
       145                93.16          󰁹          S     100.00
       146                94.26          󰁹          S     100.00
       147                97.32          󰥔          S     100.00
       148                95.79          󰥔          S     100.00
       149                97.67          󰥔          S     100.00
       150               100.00          󰥔          S     100.00
       151               100.00          󰥔          S     100.00
       152                68.11          󰥔          B      80.13
       153               100.00          󰥔          S     100.00
       154               100.00          󰥔          S     100.00
       155               100.00          󰥔          S     100.00
       156               100.00          󰥔          S     100.00
       157               100.00          󰥔          S     100.00
       158               100.00          󰥔          S     100.00
       159               100.00          󰥔          S     100.00
       160               100.00          󰥔          S     100.00
       161               100.00          󰥔          S     100.00
       162               100.00          󰥔          S     100.00
       163               100.00          󰥔          S     100.00
       164               100.00          󰥔          S     100.00
       165               100.00          󰥔          S     100.00
       166               100.00          󰥔          S     100.00
       167               100.00          󰥔          S     100.00
       168               100.00          󰥔          S     100.00
       169               100.00          󰥔          S     100.00
       170               100.00          󰥔          S     100.00
       171               100.00          󰥔          S     100.00
       172               100.00          󰥔          S     100.00
       173               100.00          󰥔          S     100.00
       174               100.00          󰥔          S     100.00
       175               100.00          󰥔          S     100.00
       176               100.00          󰥔          S     100.00
       177               100.00          󰥔          S     100.00
       178               100.00          󰥔          S     100.00
       179               100.00          󰥔          S     100.00
       180                68.11          󰥔          B      80.13
       181                94.97          󰥔          S     100.00
       182                99.95          󰥔          S     100.00
       183                66.74          󰥔          B      78.52
       184                67.31          󰥔          B      79.18
       185                79.88          󰥔          A      93.97
       186                99.80          󰥔          S     100.00
       187                71.18          󰥔          B      83.74
       188                68.02          󰥔          B      80.02
       189                76.30          󰥔          B      89.77
       190                61.23          󰥔          B      72.03
       191                99.84          󰥔          S     100.00
       192                20.92          󰥔          F      24.62
       193                84.87          󰥔          A      99.85
       194                84.96          󰥔          A      99.95
       195                96.20          󰁹          S     100.00
       196                28.96          󰥔          D      34.07
       197                41.94          󰥔          D      49.34
       198                37.72          󰥔          D      44.38
       199                37.59          󰥔          D      44.22
       200                31.11          󰥔          D      36.60
       201                21.18          󰥔          F      24.92
       202                26.48          󰁹          D      31.15
       203                29.06          󰁹          D      34.19
       204                25.47          󰁹          F      29.96
       205                28.72          󰁹          D      33.79
       206                37.03          󰁹          D      43.56
       207                37.03          󰁹          D      43.56
       208                22.73          󰁹          F      26.74
       209                33.84          󰁹          D      39.81
       210                33.34          󰁹          D      39.22
       211                26.48          󰁹          D      31.15
       212                19.27          󰁹          F      22.67
       213                16.76          󰁹          F      19.72
       214                79.30          󰁹          A      93.29
       215                36.08          󰁹          D      42.45
       216                37.95          󰥔          D      44.65
       217                31.09          󰥔          D      36.58
       218                31.09          󰥔          D      36.58
       219                12.34          󰥔          F      14.52
       220                13.14          󰥔          F      15.46
       221                11.64          󰥔          F      13.69
       222                13.02          󰥔          F      15.32
       223                14.80          󰥔          F      17.41
       224                14.90          󰥔          F      17.53
